Description: Disables the Java version check since the JRE in Debian always
 meets the minimum requirement and the check routinely breaks with the latest JREs.
Author: Emmanuel Bourg <ebourg@apache.org>
Forwarded: not-needed
--- a/make/jogamp-env.xml
+++ b/make/jogamp-env.xml
@@ -45,21 +45,6 @@
       </condition>
     </fail>
 
-    <fail message="Unsupported Java version: ${ant.java.version}. Make sure that the version of the Java compiler is 1.6 (6.0) or greater.">
-        <condition>
-            <not>
-                <or>
-                    <equals arg1="${ant.java.version}" arg2="1.6"/>
-                    <equals arg1="${ant.java.version}" arg2="1.7"/>
-                    <equals arg1="${ant.java.version}" arg2="1.8"/>
-                    <equals arg1="${ant.java.version}" arg2="1.9"/>
-                    <equals arg1="${ant.java.version}" arg2="2.0"/>
-                    <equals arg1="${ant.java.version}" arg2="2.1"/>
-                </or>
-            </not>
-        </condition>
-    </fail>
-
   </target>
 
   <target name="jogamp.env.init" depends="jogamp.env.validation">
